.github/workflows/rerun_workflow_on_push.yml

```diff
@@ -0,0 +1,79 @@
+---
+name: Rerun affected workflows in long lived branches on push
+
+on:
+  push:
+    branches:
+      - main
+      - candlepin-*-HOTFIX
+
+jobs:
+  rerun-workflows:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Mask secrets
+        shell: bash
+        run: |
+          echo "::add-mask::${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Get PR info
+        uses: 8BitJonny/gh-get-current-pr@fa2160116055f6fe10376375892b255f8b2ed8f6
+        id: PR
+        with:
+          github-token: ${{ github.token }}
+
+      - name: Rerun workflows
+        if: ${{ !contains(steps.PR.outputs.pr_labels, 'skip CI') }}
+        shell: bash
+        run: |
+          base_branch="${GITHUB_REF#refs/heads/}"
+          
+          # List affected PRs exclude dependabots, weblate and draft PRs
+          pr_head_shas=$(gh api \
+          -H "Accept: application/vnd.github+json" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          /repos/candlepin/candlepin/pulls | \
+          jq --arg base "$base_branch" -r '.[] | select(.base.ref == $base and .draft == false and .user.login != "weblate" and .user.login != "dependabot[bot]") | .head.sha')
+
+          # List runner ids from PRs(only for pr_verification.yml)
+          for head_sha in $pr_head_shas; do
+            runner_id=$(gh api \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            /repos/candlepin/candlepin/actions/workflows/pr_verification.yml/runs?head_sha=$head_sha | \
```

Comment:
This will need to be updated after the pr_verification.yml workflow gets split into many files.

Reply:
yes